vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -1997,4 +1997,9 @@ - (void)unblindedTokensReady
   }
 }
 
+- (void)deleteUnblindedTokensForPromotion:(const std::string&)promotion_id callback:(ledger::ResultCallback)callback
+{
+  // TODO please implement
```

Comment:
@kylehickinson can you please add implementation of it, you can find reference for it here https://github.com/brave/brave-core/pull/4035/files#diff-804d4c26e5326ea7739bb961bd6d5dfcR168

Reply:
@NejcZdovc Is there a reason to do this? iOS build on master branch already does not function correctly as we are waiting on #4087

Reply:
that's true, we will migrate to ledger anyway. Are you happy with just this comment for now?

Reply:
Yep, this is fine :)